packages-legacy/playwright/project.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "playwright-legacy",
```

Comment:
Probably we don't want to make a 'legacy' version of this plugin, but the current publish process adds this as a dep to the actual plugin. Just added to get working, but probably should update the publish process to exclude certain plugins?

Reply:
I think it's probably fine for now. We'll remove it in Nx 17.